src/status_im2/contexts/chat/messages/content/reactions/view.cljs

```diff
@@ -1,12 +1,11 @@
 (ns status-im2.contexts.chat.messages.content.reactions.view
   (:require [status-im2.constants :as constants]
             [quo2.core :as quo]
-            [react-native.core :as rn]
             [utils.re-frame :as rf]
             [status-im2.contexts.chat.messages.drawers.view :as drawers]))
 
 (defn- on-press
-  [own message-id emoji-id emoji-reaction-id]
+  [{:keys [own message-id emoji-id emoji-reaction-id]}]
```

Comment:
👍 

also, do you know what `own` is ? it's a pretty vague prop name imo

Reply:
It's whether the reaction is "owned" by the user i.e. pressed in. Don't like it either, would rather have a more generic `pressed?`, but that's how it comes from the subscription and didn't want to introduce more changes than needed.

Reply:
if it's not too much work can we at least change it on the quo2 ui component? 🤔